app/src/pages/common/settingsPage/defectTypesTab/defectTypesGroup.jsx

```diff
@@ -79,7 +89,10 @@ export class DefectTypesGroup extends Component {
         <div className={cx('defect-type-group-footer')}>
           <GhostButton
             icon={PlusIcon}
-            disabled={group.length >= this.MAX_DEFECT_SUBTYPES_COUNT}
+            disabled={
```

Comment:
I think we should hide this button also with its right caption and `defectSubType` controls (see the screenshot below)
![image](https://user-images.githubusercontent.com/31861805/58575260-5cf10180-824a-11e9-9e04-d63f23c7b24b.png)

Reply:
According to our [4th](http://dev.epm-rpp.projects.epam.com:8080/ui/#superadmin_personal/settings/defect) version (see the screenshot below)
![image](https://user-images.githubusercontent.com/31861805/58575374-945fae00-824a-11e9-86ac-688db03fc2f9.png)